tracker_test.go

```diff
@@ -79,24 +101,23 @@ func TestTrackerListItemsNotNil(t *testing.T) {
 			t.Error("Returned Tracker found nil")
 		}
 	}
-	test.DeleteTrackerOK(t, nil, nil, &controller, item1.ID)
-	test.DeleteTrackerOK(t, nil, nil, &controller, item2.ID)
 }
 
 // This test ensures that ID returned by Show is valid.
 // refer : https://github.com/almighty/almighty-core/issues/189
 func TestCreateTrackerValidId(t *testing.T) {
 	resource.Require(t, resource.Database)
+	defer gormsupport.DeleteCreatedEntities(DB)()
+
 	controller := TrackerController{Controller: nil, db: gormapplication.NewGormDB(DB), scheduler: RwiScheduler}
-	payload := app.CreateTrackerAlternatePayload{
+	payload := getTrackerPayload(trackerAttr{
 		URL:  "http://issues.jboss.com",
 		Type: "jira",
-	}
+	})
 	_, tracker := test.CreateTrackerCreated(t, nil, nil, &controller, &payload)
-
-	_, created := test.ShowTrackerOK(t, nil, nil, &controller, tracker.ID)
-	if created != nil && created.ID != tracker.ID {
-		t.Error("Failed because fetched Tracker not same as requested. Found: ", tracker.ID, " Expected, ", created.ID)
```

Comment:
This comparison block could look tidier if written as `assert.Equal(v1,v2,errorString)`